src/components/images/attributes.js

```diff
@@ -0,0 +1,158 @@
+/**
+* Copyright 2012-2016, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+var cartesianConstants = require('../../plots/cartesian/constants');
+
+
+module.exports = {
+    _isLinkedToArray: true,
+
+    source: {
+        valType: 'string',
+        role: 'info',
+        description: [
```

Comment:
Yep works fine!

I'm not 100% sure if there's a possibility for XSS here (or if it should be something that _we_ worry about in plotly.js) but I couldn't manage to break anything or set any code to be executed - maybe I'm just not a good enough h4X0r though.